### PR TITLE
added more details to log msg in models.py

### PR DESCRIPTION
**Pull request by hansmannj**
added collection, item and asset to the extra
added size of asset file to log msg

### CODE REVIEW

File: app/stac_api/models.py [hunk 1]
```diff
@@ -510,7 +510,14 @@ def upload_asset_to_path_hook(instance, filename=None):
     Returns:
         Asset file path to use on S3
     '''
-    logger.debug('Start computing asset file %s multihash', filename)
+    logger.debug(
+        'Start computing asset file %s multihash (file size: %s MB)',
+        filename,
+        int(instance.file.size / 1024**2),
+        extra={
+            "collection": instance.item.collection, "item": instance.item, "asset": instance.name
```
Comment (ltshb) [inline app/stac_api/models.py:518]: `instance.item.collection` return the object not the name, same for `instance.item` ! It need to be `instance.item.collection.name` and `instance.item.name`.

**Review by ltshb — CHANGES_REQUESTED** [review]
Also add the `extra` to the log in line 529 for consistency.

File: app/stac_api/models.py (hunk 1 above)
Comment (hansmannj) [reply app/stac_api/models.py:518]: :+1:

File: app/stac_api/models.py [hunk 2]
```diff
@@ -510,7 +510,16 @@ def upload_asset_to_path_hook(instance, filename=None):
     Returns:
         Asset file path to use on S3
     '''
-    logger.debug('Start computing asset file %s multihash', filename)
+    logger.debug(
+        'Start computing asset file %s multihash (file size: %s MB)',
```
Comment (ltshb) [inline app/stac_api/models.py:514]: Uses `%.1f` or `%.3f` instead of `%s`.

File: app/stac_api/models.py [hunk 3]
```diff
@@ -510,7 +510,16 @@ def upload_asset_to_path_hook(instance, filename=None):
...
+        filename,
+        int(instance.file.size / 1024**2),
```
Comment (ltshb) [inline app/stac_api/models.py:516]: Based on comment above remove the type casting to `int`

File: app/stac_api/models.py [hunk 4]
```diff
@@ -523,7 +532,12 @@ def upload_asset_to_path_hook(instance, filename=None):
         'Set uploaded file %s multihash %s to checksum:multihash; computation done in %ss',
```
Comment (ltshb) [inline app/stac_api/models.py:532]: You may also change the `%ss` to `%.3fs` here as well

**Review by ltshb — APPROVED** [review]
Just small improvements but otherwise is ok

File: app/stac_api/models.py (hunk 2 above)
Comment (hansmannj) [reply app/stac_api/models.py]: done

File: app/stac_api/models.py (hunk 4 above)
Comment (hansmannj) [reply app/stac_api/models.py]: done

File: app/stac_api/models.py (hunk 3 above)
Comment (hansmannj) [reply app/stac_api/models.py]: :+1:

**Pull request merged**